firmware/library/L1_Drivers/pin_configure.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
```

Comment:
Should put documentation at the start of the file to explain how to use this driver.

Reply:
Done.